### PR TITLE
Call parent stylesheets were missing

### PR DESCRIPTION
**Pull request by zuk3975**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | While creating example module https://github.com/PrestaShop/example-modules/pull/107 I noticed that we don't call parent() anywhere we add custom stylesheets (e.g. the orders page). That haven't raised any problems until now, because we don't have any styles added in layout, but we add them manually in every page, but it restricts the possibility to add the styles to whole layout as I tried in the module. E.g. in module Im adding my custom flash message which can be shown in any page, so I extend layout.html and add custom css into it using the module, but all the pages that overrides stylesheet block and doesn't call parent, won't get the styles adjusted.
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| Related PRs       | https://github.com/PrestaShop/example-modules/pull/107
| How to test?      | see bellow
| Possible impacts? | Some of pages might not call the parent for a reason, but I don't know if we have such?

**How to test**
Use module to test it https://github.com/PrestaShop/example-modules/pull/107 (described in PR). Also can be tested by developers by adding some css into layout.html.twig stylesheets block and checking if it is loaded in orders page.
To test this PR using the mentioned module:
1. install the module on develop branch
2. Go to orders page and open any order for viewing
3. You should see some extra texts added from the module. Pay attention to the custom flash message at the top - when on develop branch the flash will be not styled - white.
4. Now switch to my PR branch and refresh the order page.
5. You should see the flash turned to "yellow-green" color. This means my PR works because order page includes css from parent. (see screenshots how the flash should look in develop and in my PR)
with this PR changes:
Screenshot from 2022-11-12 16-38-10
without this PR changes:
Screenshot from 2022-11-12 16-37-51

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/index.html.twig
Comment (zuk3975) [inline src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/index.html.twig:29]: for now I only added it in orders page, but there are more. I can adjust the PR and add it everywhere I see (after the idea is accepted and if im not missing something here)

**Comment by zuk3975** [comment]
ping @PrestaShop/committers waiting for feedback if im not missing something, before I start adding parent calls in every template which is missing it (and maybe if anyone knows any pages where the parent() was not called intentionally)

**Comment by matks** [comment]
> ping @PrestaShop/committers waiting for feedback if im not missing something, before I start adding parent calls in every template which is missing it (and maybe if anyone knows any pages where the parent() was not called intentionally)

I think you found a good problem and that is a good idea to fix it 👍

**Review by NeOMakinG — DISMISSED** [review]
Seems legit 🐐

File: src/PrestaShopBundle/Resources/views/Admin/Exception/error.html.twig
Comment (zuk3975) [inline src/PrestaShopBundle/Resources/views/Admin/Exception/error.html.twig:43]: automatic change from storm. Apparently removed some useless spaces

**Review by NeOMakinG — DISMISSED** [review]

**Comment by zuk3975** [comment]
rebased

**Review by NeOMakinG — DISMISSED** [review]

**Review by SharakPL — DISMISSED** [review]

**Comment by SharakPL** [comment]
Why is phpunit not happy?

**Comment by ghost** [comment]
I restart CI and remove `waiting for QA `

**Comment by NeOMakinG** [comment]
It probably needs a rebase, if the CI has been fixed

**Comment by zuk3975** [comment]
rebased again :heavy_check_mark: hopefully got fixed this time :smile:

**Review by yanmakouf — APPROVED** [review]

**Review by yanmakouf — COMMENTED** [review]
Everything looks good :) 👍

**Review by NeOMakinG — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by FabienPapet — APPROVED** [review]

**Comment by zuk3975** [comment]
@sLorenzini / @sarahdib / @sallemiines Hi, I see you assigned/unassigned yourself and now there are nobody assigned to the issue, maybe you forgot to add QA :heavy_check_mark: label? Or should some developers test it manually?

**Comment by sallemiines** [comment]
Hello @zuk3975 

Im on this PR to test it. But i see that in `How to test `section, we should use this [module](https://github.com/PrestaShop/example-modules) of this related [PR](https://github.com/PrestaShop/example-modules/pull/107).
Should I test both PRs (This one and [PR2](https://github.com/PrestaShop/example-modules/pull/107)) in parallel ? Or, i can test it separetely ?

Im waiting for your feedback.

Thank You!

**Comment by zuk3975** [comment]
@sallemiines updated the PR "How to test" section. Please take a look, it should be clear now. So that way you can approve both - this PR and the module at the same time.

**Comment by florine2623** [comment]
Hello @zuk3975 ,

> 1. install the module on develop branch

I can't install the module https://github.com/PrestaShop/example-modules on develop branch, I don't see one: 
Screenshot 2022-11-22 at 09 39 54

Am I missing something? ^^

**Comment by zuk3975** [comment]
@florine2623 the module that i provided is a new PR to demomodules (so ofc its not yet in master).
You can checkout to my module branch and install that (https://github.com/PrestaShop/example-modules/pull/107/files#diff-911968c4bca3fb8ece54a91c1e0985349d2e1a9a482f5a8ccd451fdcba964173R29) module on prestashop develop branch.

**Review by florine2623 — COMMENTED** [review]
I have installed PR https://github.com/PrestaShop/PrestaShop/pull/29646 and module https://github.com/PrestaShop/example-modules/pull/107 but it won't show in the BO:

Screenshot 2022-11-22 at 16 24 10

Screenshot 2022-11-22 at 16 24 54

Screenshot 2022-11-22 at 16 29 21


And nothing is shown on the order detail page either.

**Comment by zuk3975** [comment]
@florine2623 sorry if I wasn't clear. You are almost there... :smile: 
example modules is kind of repository which holds alot of mudules, so instead of putting the whole "examplemodules" dir, you should open it and only take the demoextendtemplate module which is inside. PM me in slack if its still unclear, will be faster :)

**Review by florine2623 — APPROVED** [review]
Launched automated tests, they are OK ✅ 

As discussed directly with @zuk3975 ,

Tested https://github.com/PrestaShop/PrestaShop/pull/29646 in combination with https://github.com/PrestaShop/example-modules/pull/107.

I've followed these steps to test both PRs:

- Installed PR 29646
- Installed example-modules 107
- moved `demoextendtemplate` from `myprestashop/modules/example-modules` in the `myprestashop/modules` folder directly
- Go to `myprestashop/modules/demoextendtemplates` in terminal
- Run composer install
- Check BO Orders page

Module is well installed, the modifications are present in Orders Page:
Screenshot 2022-11-22 at 17 54 04

Screenshot 2022-11-22 at 17 24 33

So this PR is QA ✅ 

I just have a caching issue that is not related to this PR. Needs to be investigated.
If I don't delete manually my cache in `/var/cache`, I have multiple exceptions:
- In the Order detail page:
Screenshot 2022-11-22 at 17 48 20

As well as in the Orders list page:
Screenshot 2022-11-22 at 17 49 29

Once the cache is deleted, I do not have any exceptions.

This [PR](https://github.com/PrestaShop/example-modules/pull/107) is also QA ✅ as both were tested at the same time.

@zuk3975 will create the PR for caching issue ^^

Thanks @zuk3975 ! 💪

**Comment by zuk3975** [comment]
Thanks, @florine2623.
The caching issue seems to be similar to this one https://github.com/PrestaShop/PrestaShop/issues/30011 (altough its still a bit different, but you I think you can still link them in comments of your new issue).
Anyway, seems like symfony cache needs to be cleaned up manually after module uninstall (in case it used some symfony services/routes configurations including decoration (like in this pr https://github.com/PrestaShop/PrestaShop/pull/29322))

**Pull request merged**
